### DISCUSSION TITLE
Play integration

### DISCUSSION THREAD

**Discussion by jonyw4** [skiptools/skip#92] [General]
Hello there folks,
I am considering using Skip for my app. This week I got charmed by this another tool called Play. Do you know if is possible to integrate Skip with it?

https://createwithplay.com/

**Comment by aabewhite** [comment]
Sorry, I don't know enough about Play to answer. If Play outputs standard Swift and SwiftUI, then Skip could probably translate it for Android... mostly. But then you'll probably run into trouble. You see, while Skip's API support is ever-expanding, it is still limited. So if Play tends to use some Foundation or SwiftUI API that Skip doesn't yet support, you'd have to edit its output by hand every time to work around it. And I don't know how well Play would deal with you wanting to constantly make changes to the source.

**Reply by jonyw4** [reply]
No problems!  Thanks for the info. 

I've been thinking – what if we use Play kind of like Figma, to make design easier and just copy-paste Swift code from there? And then use Skip to turn that Swift code into Kotlin? Would that makes sense? What do you think about it? My goal is to reduce the overhead of my team in the crafting of our app.

**Reply by aabewhite** [reply]
I think it might work in broad strokes, but again I worry about whether Skip would support Play's output well. It all depends on what APIs Play likes to use vs. what Skip supports, how "clean" Play's output is, etc.

**Reply by jonyw4** [reply]
Okay. Thx Abe!
